packages/editor-ui/src/composables/usePushConnection.ts

```diff
@@ -423,6 +423,7 @@ export function usePushConnection({ router }: { router: ReturnType<typeof useRou
 							message: runDataExecutedErrorMessage,
 							type: 'error',
 							duration: 0,
+							dangerouslyUseHTMLString: true,
```

Comment:
seems like this could now be used for XSS since `lastNodeExecuted` on line 416 is user defined.

Reply:
Just checked and we already sanitize the message and title in show message https://github.com/n8n-io/n8n/blob/47f1dd6209381bc0f8016681cea03161b6768022/packages/editor-ui/src/composables/useToast.ts#L46

Reply:
added tests